Docs/MANUAL.md

```diff
@@ -0,0 +1,322 @@
+
+This ensures that even if someone acquires your device in an unlocked mode, they won't be able to get to your seed data.
+
+## Exporting a Seed
+
+You should be able to safely and securely use your seed within **Gordian Seed Tool** by responding to `ur:crypto-requests` and deriving keys. However, if you want to someday export the whole seed, you can.
+
+A seed can be exported by touching the "Decrypt" box under the "Data" section of a seed. This will, as usual, require your 2FA. After it decrypts, you can then click the "Share" button to the top right. This will allow you to export as hex, as BIP39 Mnemonic Words, as ByteWords, or as a `ur:crypto-seed`. (This menu is also where the functions for exporting SSKR shares and deriving and exporting keys were.)
+
+These functions will all copy the data in the appropriate form to your clipboard, allowing you to then paste it into the app of your choice. The `ur:crypto-seed` alternatively allows you to print the QR containing the `ur:crypto-seed` for your seed.
+
+> :warning: **WARNING:** Generally, you want to always keep your seed in **Seed Tool**. It is both secure and resilient in the app. There is no reason to export it. Instead, export keys as appropriate, ideally watch-only public keys or specific derived keys in response to a `ur:crypto-request` from another app.
+
+## Deleting a Seed
+
+If you're done with a seed or if you've exported it to another app or device, you may then want to delete it.
+
+Seeds can be deleted with the "Edit" function on the main page. You can immediately "Undo" it if you deleted the wrong seed, but afterward, any seed you deleted will be gone forever. Be careful!
+
+<div align="center">
+  <table border=0>
+    <tr>
+      <td>
+        <a href="https://raw.githubusercontent.com/BlockchainCommons/GordianSeedTool-iOS/shannona-minimanual/images/st-export.jpeg"><img src="https://raw.githubusercontent.com/BlockchainCommons/GordianSeedTool-iOS/shannona-minimanual/images/st-export.jpeg" width=250></a> 
+        <br><div align="center"><b>Export</b></div>
+      </center></td>
+      <td>
+        <a href="https://raw.githubusercontent.com/BlockchainCommons/GordianSeedTool-iOS/shannona-minimanual/images/st-sskr-export-2.jpeg"><img src="https://raw.githubusercontent.com/BlockchainCommons/GordianSeedTool-iOS/shannona-minimanual/images/st-delete-1.jpeg" width=250></a> 
+        <br><div align="center"><b>Delete</b></div>
+      </center></td>
+      <td>     
+        <a href="https://raw.githubusercontent.com/BlockchainCommons/GordianSeedTool-iOS/shannona-minimanual/images/st-sskr-expor-3.jpeg"><img src="https://raw.githubusercontent.com/BlockchainCommons/GordianSeedTool-iOS/shannona-minimanual/images/st-delete-2.jpeg" width=250></a> 
+        <br><div align="center"><b>Undo Delete</b></div>
+      </center></td>
+    </tr>
+  </table>
+</div>
```

Comment:
We need a section on settings.

Reply:
added new section, expanding previous bullet point.